interview-questions.md

```diff
@@ -16,42 +16,46 @@ Researched answer:
 
 Your answer: git is an information tracker program that allows more than one person to work on the same code and transfer them to one another. It also allows pople to check out branches and make changes that wont affect the main branch. Github is somewhere where peopel can upload their branches and clone repositories to share their work. We often use git and Github to share our work between our cohort. 
 
-Researched answer:
+Researched answer: git stands for Global Information Tracker, and is used to track progress between multiple computers and people at the same time. Using git you can check out branches, make changes and wreck the file with no changes to the main one. Github is a cloud sharing service, wherep eopel can upload their git repositories and share them amongst one another. It is essential for uploading and sharing work.
 
 2. Which JavaScript operators will return a Boolean value?
 
 Your answer: Operators that compare things, such as <, >, <=, >= will return a Boolean value, which is true or false. 
```

Comment:
What are the three categories of operators that give a Boolean value? You have listed the relational operators. Two more categories to go.